src/pages/documentation/release-notes/2021.md

```diff
@@ -31,18 +25,11 @@ description: Archived Release Notes for 2021
 * Adds XDM Implementation Details to each Experience Event sent to the Edge Network.
 * Fixes generic network error format to conform to EdgeEventError type so they are dispatched back to the caller correctly.
 
-### Android Core 1.9.2
-
-* Fixed a bug where event number was not incremented correctly for shared states.
-* Stability improvements for Configuration extension.
-* Configuration now allows for empty appId to reset the previously set appId value.
-* The Event Hub shares wrapper type in its shared state.
-
 ## December 20, 2021
 
 ### React Native Plugins with Swift SDKs (Beta)
 
-The following Adobe Experience Platform React Native plugins are integrated with Android SDKs and iOS **Swift** SDKs.
```

Comment:
same here, wrappers contain updates for both platforms, so let's keep this note for android